code/2_analysis/01_make_maps.R

```diff
@@ -0,0 +1,135 @@
+# Description: This file produces all maps in the paper. Thus, it generates
+# Figures 1, 2a, 2c, 2e, 3a, and Appendix Figures A1a, A1c, A4a and A4c.
```

Comment:
@cheikhfall07 the figures in the appendix don't actually have a/b/c/d labels - how hard would it be to add them? OK if not doable, it's not a high priority.

Reply:
We discussed. I think given the timeline it's fine to leave as is.